src/app/space/app-launcher/services/app-launcher-gitprovider.service.ts

```diff
@@ -119,12 +119,12 @@ export class AppLauncherGitproviderService implements GitProviderService {
             this.gitHubUserLogin = user.login;
             orgs.push(this.gitHubUserLogin);
             let gitHubDetails = {
-              authenticated: this.isPageRedirect() ? true : false,
+              authenticated: true,
               avatar: user.avatarUrl,
               login: user.login,
               organizations: orgs
             } as GitHubDetails;
-            return this.isPageRedirect() ? Observable.of(gitHubDetails) : Observable.empty();
```

Comment:
We should reomove the private isPageRedirect method now

Reply:
@dlabrecq thanks Dan, have updated it here https://github.com/fabric8-ui/fabric8-ui/pull/2701